Content.Server/Forensics/Components/TransferForensicsOnSpawnBehaviorComponent.cs

```diff
@@ -0,0 +1,12 @@
+namespace Content.Server.Forensics
+{
+    /// <summary>
+    ///     Transfers forensics from this entity to spawned objects after it's destruction
+    /// </summary>
+    [RegisterComponent]
+    public sealed partial class TransferForensicsOnSpawnBehaviorComponent : Component
```

Comment:
this is unnecessary, just add a `TransferChanceOnSpawnBehavior` field to `ForensicsComponent` (xmldoc it) and subscribe that to the event, the lack or not of the component will serve as the TransferForensics bool (could also set the prob to 0). Remember to test that forensics still work afterwards